src/modules/RA/ra-modules.js

```diff
@@ -141,6 +141,7 @@ export const config = {
     {
       label: 'baseLayer.project',
       requiredModule: 'BaseLayer',
+      // requiredPermissions: 'can_manage_baselayers',
```

Comment:
Commented code should not be present, unless for very precise reasons

Reply:
I kept this comment, as permissions could be a solution for the ticket and should be discussed with Pauline or Jeremie

Reply:
Do you want me to erase this comment ?

Reply:
Even though some change could appear in the work of this PR, commented code should not be present.